core/src/main/java/org/kohsuke/stapler/Stapler.java

```diff
@@ -96,6 +96,15 @@
  */
 public class Stapler extends HttpServlet {
 
+    /**
+     * Exceptions we don't want to print a stacktrace for because they are normal behaviour.
+     * [HUDSON-4834] A stack trace is too noisy for this; could just need to log in.
```

Comment:
```suggestion
     * [JENKINS-4834] A stack trace is too noisy for this; could just need to log in.
```

https://issues.jenkins.io/browse/JENKINS-4834 FTR